src/plugins/projectGalaxy/index.ts

```diff
@@ -0,0 +1,73 @@
+// URLS
+const API_BASE_URL = 'https://graphigo.stg.galaxy.eco/query';
+// const API_BASE_URL = 'https://graphigo.prd.galaxy.eco/query';
+
+export default class Plugin {
+  public author = 'ProjectGalaxy';
+  public version = '1.0.0';
+  public name = 'Project Galaxy Module';
+  public options: any;
```

Comment:
This is all unused and it's an approach from the old plugin system that should not be copied. This info is all taken from the plugin's plugin.json.